### ISSUE TITLE
Always refresh log "have not found myself in list yet"

### ISSUE THREAD

**Issue by luohua13** [kmodules/peer-finder#17]
Why "newPeers.Equal(peers)" is hitting refresh the log about "have not found myself in list yet" ?

**Comment by yelnyleenew** [comment]
yes, I also had this question, this message give us some information that current nacos node not in nacos cluster, 
In fact, it should indicate that the current node is in this cluster.
